pkg/antctl/raw/set/flow-aggregator/command.go

```diff
@@ -0,0 +1,167 @@
+		},
+		"clickHouse.debug": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setBoolOrFail(&c.ClickHouse.Debug, value)
+		},
+		"clickHouse.compress": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			if c.ClickHouse.Compress == nil {
+				c.ClickHouse.Compress = new(bool)
+			}
+			return setBoolOrFail(c.ClickHouse.Compress, value)
+		},
+		"clickHouse.commitInterval": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setCommitIntervalOrFail(&c.ClickHouse.CommitInterval, value)
+		},
+		"flowCollector.enable": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setBoolOrFail(&c.FlowCollector.Enable, value)
+		},
+		"flowCollector.address": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setStringOrFail(&c.FlowCollector.Address, value)
+		},
+	}
+	return Command
+}
+
+func updateRunE(cmd *cobra.Command, args []string) error {
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	k8sClient, _, err := raw.SetupClients(kubeconfig)
+	if err != nil {
+		return fmt.Errorf("failed to create clientset: %w", err)
+	}
+	configMapName := os.Getenv("FA_CONFIG_MAP_NAME")
+	configMap, err := util.GetFAConfigMap(k8sClient, configMapName)
+	if err != nil {
+		return err
+	}
+	var flowAggregatorConf flowaggregatorconfig.FlowAggregatorConfig
+	if err := yaml.Unmarshal([]byte(configMap.Data["flow-aggregator.conf"]), &flowAggregatorConf); err != nil {
+		return err
+	}
+	for _, query := range args {
+		pair := strings.Split(query, "=")
+		if len(pair) != 2 {
+			return fmt.Errorf("query should contains exactly one '='")
+		}
+		mutatorFn, ok := mutators[pair[0]]
+		if !ok {
+			return fmt.Errorf("unknown configuration parameter, please check antctl set -h")
+		}
+		if err := mutatorFn(&flowAggregatorConf, pair[1]); err != nil {
+			return err
+		}
+	}
+	// marshal back the changed parameters to configmap
+	b, err := yaml.Marshal(&flowAggregatorConf)
+	if err != nil {
+		return err
+	}
+	configMap.Data["flow-aggregator.conf"] = string(b)
+	if _, err := k8sClient.CoreV1().ConfigMaps("flow-aggregator").Update(context.TODO(), configMap, metav1.UpdateOptions{}); err != nil {
+		return err
+	}
+	return nil
+}
+
+func setBoolOrFail(b *bool, value string) error {
+	boolValue, err := strconv.ParseBool(value)
+	if err != nil {
+		return err
+	}
+	*b = boolValue
+	return nil
+}
+
+func setStringOrFail(b *string, value string) error {
+	*b = value
+	return nil
+}
+
+func setCommitIntervalOrFail(b *string, value string) error {
+	commitInterval, err := time.ParseDuration(value)
+	if err != nil {
+		return err
+	}
+	if commitInterval < flowaggregatorconfig.MinClickHouseCommitInterval {
+		return fmt.Errorf("commitInterval %s is too small: shortest supported interval is %s\n", commitInterval, flowaggregatorconfig.MinClickHouseCommitInterval)
```

Comment:
you should not need to add a newline character at the end of an error message
the newline character should be added when the error is ultimately logged, or printed